test/README.md

```diff
@@ -39,12 +39,12 @@ Use the `ZeebeTestRule` in your test case to start an embedded broker and client
 
 ```java
 import io.zeebe.client.ZeebeClient;
-import io.zeebe.client.api.response.WorkflowInstanceEvent;
+import io.zeebe.client.api.response.ProcessInstanceEvent;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 
-public class WorkflowTest {
+public class ProcessTest {
```

Comment:
Did we check that this still runs properly?

Reply:
I fixed the example, it was broken since December 2018 https://github.com/zeebe-io/zeebe/commit/453c2159a0211c4c8fce55b65aff45b42fe18e17